x/staking/spec/01_state.md

```diff
@@ -15,6 +15,21 @@ Store entries prefixed with "Last" must remain unchanged until EndBlock.
 
 - LastTotalPower: `0x12 -> ProtocolBuffer(sdk.Int)`
 
+## ValidatorUpdates
+
+ValidatorUpdates contains the validator updates returned to ABCI at the end of every block. 
+The values are overwritten in every block. 
+
+- ValidatorUpdates `0x51 -> []abci.ValidatorUpdate`
+
+## UnbondingId
+
+UnbondingId stores the ID of the latest unbonding operation. It enables to create unique IDs for 
```

Comment:
sanity check: should the unbonding ID be something like [delegatorAddress|blockheight] or does it matter?

Reply:
At the moment it's just a sequence number that gets incremented. For the system it's irrelevant as long as it's unique.